pkg/pillar/zedcloud/send.go

```diff
@@ -377,6 +377,15 @@ func VerifyAllIntf(ctx *ZedCloudContext, url string, requiredSuccessCount uint,
 			verifyRV.RemoteTempFailure = true
 		}
 		if err != nil {
+			var noAddrErr *types.IPAddrNotAvailError
+			if errors.As(err, &noAddrErr) {
```

Comment:
isn't `errors.Is` enough?

Reply:
I need to compare error by type, not by value.

Reply:
Thx for the explanation!